javascripts/main.js

```diff
@@ -4,21 +4,25 @@
     compiledtemplateFn = null,
     projectsPanel = null;
 
-  var renderProjects = function (tags, names) {
+  var renderProjects = function (tags, names, page) {
+    var tagsString = tags;
+    tags = (tags || "").toLowerCase().split(",")
```

Comment:
As far as I can tell, `selectedTags` is never actually used, so I propose aligning `tags` usage with `names`:

1. `tags` variable remains the `,`-delimited list.
2. `tags` split is deferred to [`projectsSvc.applyTagsFilter`](https://github.com/up-for-grabs/up-for-grabs.net/blob/8f2cabb8b9e86733b5f609351c28c82f9f3d0d81/javascripts/projectsService.js#L2-L5), which already handles it coming through un-split.
3. `selectedTags` can then be used instead of adding `tagsString`

You might check `blame` to make sure I'm not overlooking some reason why the `tags` split needed to happen where it did before you moved it here.